### PR TITLE
[Jhosie] Fix issues with UI and current repo

### PR DESCRIPTION
**Pull request by wiggle131**
**What does this PR do?**
Implements Login and Home UI to current Repo
**Where should the reviewer start?**

**Related ticket link/s:**

**Before and After UI Screenshot (optional):**

### CODE REVIEW

**Comment by wiggle131** [comment]
Fix Animals button to redirect to Animals table instead of plants

File: client/src/pages/Home/index.tsx [hunk 1]
```diff
@@ -50,8 +51,8 @@ export default function Home() {
   };
 
   async function show() {
-    const blob = await getFile(axios, 'tobacco.obj');
-    downloadBlob(blob, 'tobacco.obj');
+    const blob = await getFile(axios, 'leaf.obj');
+    downloadBlob(blob, 'leaf.obj');
```
Comment (e6654321) [inline client/src/pages/Home/index.tsx:55]: you can delete this since this is for testing only

**Review by e6654321 — APPROVED** [review]
erase function in home

File: client/src/pages/Login/login.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.center {
```
Comment (e6654321) [inline client/src/pages/Login/login.css:1]: You can also mvoe this to index.css, para wla nay .css files sa mga folder lain

Comment (wiggle131) [reply client/src/pages/Login/login.css:1]: should I move the home.css file to index.css as well?

File: client/src/pages/Home/index.tsx [hunk 3]
```diff
@@ -64,44 +65,69 @@ export default function Home() {
...
+              style={{
+                backgroundColor: '#A6E3A1',
+                fontSize: '32px',
+                height: 'auto',
+                margin: 'auto',
+                paddingLeft: '2rem',
+                paddingRight: '2rem',
+              }}
+              type="dashed"
+              onClick={handleGoToPlants}
```
Comment (AldrenKim) [inline client/src/pages/Home/index.tsx:83]: Change ang name sa function on onclick to handleGoToAnimals.

**Review by AldrenKim — APPROVED** [review]

**Pull request merged**